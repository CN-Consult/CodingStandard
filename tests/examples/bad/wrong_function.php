<?php
/**
 *
 * @file
 * @version 2.11
 * @copyright 2020 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */

/**
 * Test function
 * @param $a
 */
function test($a,$b) {
	return"1$a$b";
}

$a = test(1,3);
