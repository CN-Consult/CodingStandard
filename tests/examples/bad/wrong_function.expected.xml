<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/wrong_function.php" errors="4" warnings="0" fixable="2">
        <error line="10" column="1" source="..Functio.MissingParamTag" severity="5" fixable="0">Doc comment for parameter "$b" missing</error>
        <error line="12" column="4" source="..Functio.MissingParamComment" severity="5" fixable="0">Missing parameter comment</error>
        <error line="14" column="22" source="PEAR.Functions.FunctionDeclaration.BraceOnSameLine" severity="5" fixable="1">Opening brace should be on a new line</error>
        <error line="15" column="5" source="Generic.WhiteSpace.LanguageConstructSpacing.Incorrect" severity="5" fixable="1">Language constructs must be followed by a single space; expected "return "1$a$b"" but found "return"1$a$b""</error>
    </file>
</phpcs>
